CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+## 0.2.0 (2017-08-29)
+
+**Major Enhancements:**
+
+- Added Highstock feature of HighCharts plotting: #89 (by @Prakriti-nith)
+- Added HighMaps feature of HighCharts plotting: #92 (by @Prakriti-nith)
+- Added Custom Styling CSS feature of HighCharts plotting: #93
+(by @Prakriti-nith)
+- Exporting HighCharts to different formats: #94 (by @Prakriti-nith)
+- Added ChartWrapper feature of GoogleCharts: #95 (by @Prakriti-nith)
+- Import data from google spreadsheet: PR #88 (by @Prakriti-nith)
+- Added ChartEditor feature of GoogleCharts plotting: #96 (by @Prakriti-nith)
+- Multiple Charts having different (or same plotting library)
+in a single row(or in single cell in IRuby notebook): #97 (by @Prakriti-nith)
+- Different formatters feature of GoogleCharts: #110 (by @Prakriti-nith)
+- Require dependent JS for the plotting libraries
+GoogleCharts/HighCharts/DataTables in Rails application.js file: #115
+(by @Prakriti-nith)
+
+
+**Minor Enhancements:**
```

Comment:
In #104 methods `export_html_file` and `generate_html` have been added.
Also, the feature to load large set of data in datatables has been added in daru-data_tables. Does that need to be mentioned too?

Reply:
I added it. Thanks @Prakriti-nith